packages/theme_tailor/example/lib/build_yaml_themes.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/material.dart';
+import 'package:theme_tailor_annotation/theme_tailor_annotation.dart';
+
+part 'build_yaml_themes.tailor.dart';
+
+@tailor
+class $_AnotherTheme {
+  static List<Color> themes = [Colors.amber, Colors.blueGrey.shade800];
```

Comment:
@Rongix Shouldn't we throw error if list is shorter than number of themes?

Reply:
We should throw an error. I will create a task for it